extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -376,8 +370,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       }
     }
 
-    /** @private {?CONSENT_POLICY_STATE} */
-    this.consentState_ = null;
+    /** @protected {?CONSENT_POLICY_STATE} */
```

Comment:
Why is the visibility change necessary?

Reply:
It is accessed in constructSraRequest_ with at the instance as opposed to this, so better to make protected (otherwise linter warns):

instances[0].getPageParameters(instances[0].consentState)